content/docs/manuals/app-catalog/catalog.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Example Applications"
+summary: ""
+draft: false
+weight: 331
+toc: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+![header](header.png)
+Here, you'll find some examples of applications with their respective SLA template that you can use right away to test out Oakestra.
+
+## Nginx Client-Server with Load Balancing
+![Minecraft Preview](balancing.png)
+
+To test out the balancing capabilities of Oakestra, we can deploy a simple Nginx server and a client that sends requests to a Round-Robin balanced semantic IP assigned to the server. When scaling up the Nginx service, the client requests will automatically be balanced across the service's instances.
+
+{{< link-card
+ title="Nginx Client-Server Example"
+ description="create load-balanced nginx servers"
+ href="../nginx-client-server-with-load-balancing/"
+ target="_blank"
+>}}
+
+## Cloud/Edge Gaming: Minecraft 
+![Minecraft Preview](minecraft-full.png)
+
+Features:
+- 🎮 Play Minecraft from a browser, no client installation needed (thanks to [WebMC](https://github.com/michaljaz/webmc))
+- 👭 Play multiplayer locally or remotely 
+- 🖥️ Host your Minecraft server ([Openhack](https://github.com/noelbundick/minecraft-server)) and proxy. 
+- ⚙️ Scale your server instances to handle more users
+- 🛠️ Customize your deployment 
+
+{{< link-card
+ title="Minecraft Demo Repository"
+ description="github.com/oakestra/minecraft-client-server-example"
+ href="https://github.com/oakestra/minecraft-client-server-example"
```

Comment:
I don't think that's planned. As the repo might evolve (or might not) every time it would require wiki edits also for an external app. I created the Nginx pages only because we don't have repos for them...

Reply:
I see, fair enough 🙂